lib/Factory/UserFactory.php

```diff
@@ -35,10 +35,12 @@ public function from($params) {
 	}
```

Comment:
I’m getting a "Missing return statement" warning here.

Reply:
I added an explicit `return null` at the end of this method, but that's only a guess as to what this means. I'm not seeing that warning on my end. Are you getting it running `phpunit`?

Reply:
I’m seeing this in PhpStorm as an [Inconsistent return points](https://www.jetbrains.com/help/phpstorm/php-inconsistent-return-points.html) warning.

I’m not sure what the best solution is here. A `@return mixed|void` would also make the warning go away, but should we ever return `void` if we have other returns in a method?

If other methods like `UserFactory::from_id()` return `null` when no `Timber\User` could be returned, wouldn’t it make sense to be explicit and return `null` here, too?

Reply:
`::from_id` already returns explicitly...right? It just returns the result of the ternary expression.

I added some doc comments that I feel are appropriately specific. I don't care too much about appeasing PHPStorm since we're already [adding PHPStan](https://github.com/timber/timber/pull/2149) in parallel. Can't please everybody. :)